File: lab06/whale.py
#!/usr/bin/python3
import re
import sys
i = 0
j = 0
while 1:
    try:
        x = str(input())
        if sys.argv[1] in x:
            number = x.split(' ')
            i += int(number[0])
            j += 1
    except EOFError:
        print("{} observations: {} pods, {} individuals".format(sys.argv[1],j,i))
        exit()

File: lab06/whale.pl
#!/usr/bin/perl -w
$i = 0;
$c = 0;
	
while ($input = <STDIN>) {
    if ($input =~ /$ARGV[0]/) {
       @ch = split / /, $input;
       $c = $c + $ch[0];
       $i++;
    }
}
print "$ARGV[0] observations: $i pods, $c individuals\n";
